rootfs-builder/clearlinux/Dockerfile.in

```diff
@@ -11,12 +11,24 @@ RUN dnf -y update && dnf install -y \
 chrony \
 curl \
 gcc \
-git \
+gcc-c++ \
+autoconf \
+automake \
+m4 \
+cmake \
+wget \
```

Comment:
OOI, why is `wget` needed? We prefer to use `curl` if at all possible, but I don't see where a downloader is used here?